scripts/testnet/sovereignBridge/pyScripts/next_contract.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from multiversx_sdk_core import Address, AddressComputer
+
+
+def main():
+    # input arguments
+    address = Address.from_bech32(sys.argv[1])
```

Comment:
What does `next_contract` is supposed to mean?

Reply:
Just a short name for the file which will return a contract address computed for some nonce.

Reply:
Should be called like that. ComputeSCAddressByUserNonceAndAddress